modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_set/aah_angle_set.cc

```diff
@@ -116,36 +114,35 @@ AAH_AngleSet::ReceiveDelayedData()
 }
 
 const double*
-AAH_AngleSet::PsiBndry(uint64_t bndry_map,
-                       unsigned int angle_num,
-                       uint64_t cell_local_id,
-                       unsigned int face_num,
-                       unsigned int fi,
-                       int g,
-                       size_t gs_ss_begin,
-                       bool surface_source_active)
+AAH_AngleSet::PsiBoundary(uint64_t bndry_map,
+                          unsigned int angle_num,
+                          uint64_t cell_local_id,
+                          unsigned int face_num,
+                          unsigned int fi,
+                          int g,
+                          size_t gs_ss_begin,
+                          bool surface_source_active)
 {
-  if (ref_boundaries_[bndry_map]->IsReflecting())
-    return ref_boundaries_[bndry_map]->HeterogeneousPsiIncoming(
+  if (boundaries_[bndry_map]->IsReflecting())
+    return boundaries_[bndry_map]->PsiIncoming(
       cell_local_id, face_num, fi, angle_num, g, gs_ss_begin);
```

Comment:
That's a bit deeper than I've gotten in the code right now. Since I don't yet have a complete understanding on how reflecting boundaries are handled, I'm going to leave this here for now. I will say this is just the first pass on refactoring. This is more or less the 30,000 ft view refactoring. Just intended to clean things up before the serious work begins. :) I will look at this though... I expect there will be a lot more substantive changes in the next round.